CollapseLauncher/XAMLs/MainApp/MainPage.xaml.cs

```diff
@@ -157,6 +157,17 @@ private async void StartRoutine(object sender, RoutedEventArgs e)
                 LoadGamePreset();
                 SetThemeParameters();
 
+                if (!IsPreview)
+                {
+                    VersionNumberIndicator.Text = AppCurrentVersion.VersionString;
+                }
+                else
+                {
+                    // Temp "*" indicator for distinguishing preview builds from stable      
+                    VersionNumberIndicator.Text = AppCurrentVersion.VersionString + "*"; 
```

Comment:
Instead of just `*`, as we discussed in Discord, its probably better to shows something like `- PRE` or `- Preview`. Though I kinda like `- PRE` better as its shorts and enough for us to see that its a preview build with just one glance over a screenshot

Reply:
My concern is that the version string is already 6 characters long, adding ` - PRE` (+ 6 characters)  or ` - Preview` (+ 10 characters) might not look aesthetically pleasing for most users. 

Here's how the changes look:

- Only version number: 
![image](https://github.com/neon-nyan/Collapse/assets/9833218/cbbaf5e8-bf39-4370-a5d6-4b0108fafb10)
![image](https://github.com/neon-nyan/Collapse/assets/9833218/ad3b1f17-3bd0-48ac-93e4-283f656d8f68)

- With `- PRE`: 
![image](https://github.com/neon-nyan/Collapse/assets/9833218/e890ede7-5982-44da-93a5-07eab0c092b7)
![image](https://github.com/neon-nyan/Collapse/assets/9833218/d132e1bd-577e-4d6f-94ba-4bc6ba495b6f)

- With `- Preview`:
![image](https://github.com/neon-nyan/Collapse/assets/9833218/550fce62-6e5b-46b6-acf4-2098e288cc4c)
![image](https://github.com/neon-nyan/Collapse/assets/9833218/b9a916fb-e3fa-49f4-9756-fea5eeaa3bfc)

Reply:
Personally I'm fine with `- PRE`, but I can't say much about other peeps.

Though after much thinking, I think this isn't needed after all, as when we eventually sync stable with main/preview, it will have the same version number, so Preview and Stable version number never collapse (pun intended) together when its on different set of commits.